tools/pipelines/templates/build-npm-package.yml

```diff
@@ -39,18 +39,6 @@ parameters:
   default:
   - nyc
 
-- name: taskBundleAnalysis
-  type: boolean
-  default: false
-
-- name: taskPublishBundleSizeArtifacts
-  type: boolean
-  default: false
-
-- name: taskPack
```

Comment:
The only thing overriding `taskPack` to false was the bundle size pipeline, so there's no need for it anymore.